runtime/doc/lua.txt

```diff
@@ -1300,7 +1303,16 @@ cmd({command})                                                     *vim.cmd()*
                        autocmd FileType c setlocal cindent
                      augroup END
                    ]])
-                   vim.cmd({ cmd = 'echo', args = { '"foo"' } })
+
+                   vim.cmd { cmd = 'echo', args = { '"foo"' } }
```

Comment:
Maybe using a command that is not echo here could make the examples clearer. The double quotes of `'"foo"'` might confuse new users.

Reply:
It is a little confusing, but it demonstrates how string literals need to be passed in. I think once the user compares this to the other examples, it should be clear enough.